pom.xml

```diff
@@ -268,6 +268,12 @@
                 <groupId>org.apache.avro</groupId>
                 <artifactId>avro</artifactId>
                 <version>${avro.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.google.guava</groupId>
+                        <artifactId>guava</artifactId>
```

Comment:
Just curious, why do we need this? Also do you know where/what `{avro.version}` is defined as, I don't see it in the list of properties with all the other versions above

Reply:
avro.version comes from `confluent.common`
I asked tolga to cherry-pick this upgrade from 5.3.x because I suspected that naive version bump might break the old version (look at the original PR linked #2544)

Reply:
I think the bigger question here is why do we have different versions of the same dependencies scattered throughout the Confluent codebase. We should just update it in `confluentinc/common` and pull it in that way, so we don't end up with potential runtime errors when various confluent repos start pulling in different versions of shared dependencies and overriding each other.

Note that `guava` is already on `30.1.1-jre` on the 5.2.x branch of `confluentinc/common` (cc @wcarlson5 )

Reply:
Interesting. for 5.3.x We are planning on removing the version override and inheriting from the parent (which is also 30.1.1)

Reply:
Yeah, we should do the same thing on 5.2.x as we have for 5.3.x, unless there is something in 5.2.x which is not compatible with the newer version of guava. Is that the case or was this a preemptive exclusion? Imo we should at least try this first and see if things break. It was not clear to me whether or not this exclusion was necessary back in [#2544](https://github.com/confluentinc/ksql/pull/2544) either

Reply:
OK let's split this up. For now we know we can merge this with just cursory "does it build" testing and it will be fine. 
Then any changes removing the overrides we can pint merge when they land in 5.3.x

Reply:
This also means that if someone is on 5.3.x they won't need to bump guava in order to get CVE fixes